### PR TITLE
Add cert_dir option to kube-apiserver

### PR DESCRIPTION
**Pull request by nak3**
Add cert_dir option to kube-apiserver.

### CODE REVIEW

File: cmd/kube-apiserver/app/server.go [hunk 1]
```diff
@@ -143,6 +146,7 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 		"If HTTPS serving is enabled, and --tls_cert_file and --tls_private_key_file are not provided, "+
 		"a self-signed certificate and key are generated for the public address and saved to /var/run/kubernetes.")
 	fs.StringVar(&s.TLSPrivateKeyFile, "tls_private_key_file", s.TLSPrivateKeyFile, "File containing x509 private key matching --tls_cert_file.")
+	fs.StringVar(&s.CertDirectory, "cert_dir", s.CertDirectory, "The directory where the TLS certs are located (by default /var/run/kubernetes)")
```
Comment (kelseyhightower) [inline cmd/kube-apiserver/app/server.go]: I wonder if we should call out the fact the `cert_dir` flag is ignored if `tls_private_key_file` or `tls_cert_file` flags are set?

**Comment by nak3** [comment]
@kelseyhightower  Thank you! I agree with you.

**Comment by brendandburns** [comment]
needs gofmt.

**Comment by wojtek-t** [comment]
LGTM - merging.

**Pull request merged**